django_mongodb/fields/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .auto import MongoAutoField
+from .json_field import register_fields
```

Comment:
Use `json.py` rather than `json_field.py` (we don't need a `_field` suffix on all files in this module).